Q2.cpp

```diff
@@ -0,0 +1,83 @@
+//CodeU: Q2
+//
+//  Created by Julia Semavina on 18.05.2017
+//
+//  problem specification:
+//      Implement an algorithm to find the kth to last element of a singly linked list
+//
+//  comments:
+//      list length is unknown
+//      if K is bigger than the size of the list the answer will be IMPOSSIBLE INPUT
+//      KthToLast function contains a solution
+//
+
+#include <iostream>
+#include <vector>
+
+
+struct Node {
+    char value;
+    Node* next;
+};
+
+Node* CreateSinglyLinkedList() {
+    Node *Head = new Node;
+    std::cin >> Head->value;
```

Comment:
With all the inputs from the user it'll be nice to have some kind of prompt on what to enter when.